### DISCUSSION TITLE
No need to show the line 'Press ENTER to continue...'

### DISCUSSION THREAD

**Discussion by yurivict** [alandefreitas/matplotplusplus#222] [Ideas]
With the gnuplot backend the ```show()``` function shows the window with the plot, and also displays the message ```Press ENTER to continue...``` which expects ENTER to be pressed.

There is no need to require the user to make 2 clicks to move on. Just closing the window should be sufficient.

**Comment by alandefreitas** [comment]
> Just closing the window should be sufficient

That's not possible with Gnuplot because pipes are unidirectional.
